Bottom: 4c02390b79f095399b5e84f953485922d38d79dc
Top:    3d5acd01b190a9a227e50dfd3209e25287591137
Author: Ian Kent <ikent@redhat.com>
Date:   2013-07-13 14:18:05 +0800

Refresh of autofs-5.0.7-add-after-sssd-dependency-to-unit-file.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 7749f01..2734fe3 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -55,6 +55,7 @@
 - fix add null check in parse_server_string().
 - don't override LDFLAGS in make rules.
 - fix a couple of compiler warnings.
+- add after sssd dependency to unit file.
 
 25/07/2012 autofs-5.0.7
 =======================
